src/modules/single-course/SingleCourse.test.tsx

```diff
@@ -2,52 +2,36 @@ import { render, screen } from '@testing-library/react';
 import { useLoaderData, useNavigation } from 'react-router-dom';
 import { Mock, MockedFunction, vi } from 'vitest';
 
+import { fakeCourse } from '../../tests/__mocks__/course-data';
```

Comment:
I guess if you decided to use word "mock" instead "fake". You also should rename these variables

Reply:
Done!